src/main/java/com/github/krzysiekjodlowski/ox/Players.java

```diff
@@ -0,0 +1,61 @@
+package com.github.krzysiekjodlowski.ox;
+
+import com.github.krzysiekjodlowski.ox.model.Symbol;
+
+/**
+ * Container for two players, used
+ * mainly for managing their turns.
+ *
+ * @author Krzysztof Jodlowski
+ */
+class Players implements Subscriber {
+  private final Player playerOne;
+  private final Player playerTwo;
+  private Player currentPlayer;
+
+  private Players(Builder builder) {
+    this.playerOne = builder.playerOne;
+    this.playerTwo = builder.playerTwo;
+    this.currentPlayer = builder.startingPlayer;
+  }
```

Comment:
You have only three parameters, all of them required. Do you need builder for this?

Reply:
Not for now, but there will be also option for setting the name of each player:)